mods/cnc/maps/shellmap/map.yaml

```diff
@@ -994,6 +994,9 @@ Rules:
 			Effect: Desaturated
 		LuaScript:
 			Scripts: shellmap.lua
+		MusicPlaylist:
+			StartingMusic: map1
```

Comment:
Actually, it seems to be ignoring this completely, and picking a random track at startup.